recipes/pantax/build.sh

```
@@ -0,0 +1,19 @@
+else
+    cd gurobi11
+    ${PYTHON} -m pip install gurobipy-11.0.2-cp310-cp310-manylinux2014_x86_64.manylinux_2_17_x86_64.whl
+    rm gurobipy-11.0.2-cp310-cp310-manylinux2014_x86_64.manylinux_2_17_x86_64.whl
+    mkdir -p ${PREFIX}/bin/tools
+    cd ../vg
+    cp vg* ${PREFIX}/bin/tools/vg
+    cd $SRC_DIR/scripts
+    chmod +x pantax
+    cp $SRC_DIR/scripts/pantax ${PREFIX}/bin
+    cp $SRC_DIR/scripts/*py ${PREFIX}/bin
+    cd $SRC_DIR/tools/fastix
```

Shellcheck

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)